### PR TITLE
Add appropriate citations in the README

### PR DESCRIPTION
**Pull request by Ashutosh-Adhikari**

### CODE REVIEW

File: common/evaluators/relevance_transfer_evaluator.py [hunk 1]
```diff
@@ -0,0 +1,76 @@
+import warnings
```
Comment (daemon) [inline common/evaluators/relevance_transfer_evaluator.py:1]: Why is this in the readme PR?

File: README.md [hunk 2]
```diff
@@ -71,4 +72,22 @@ gzip -d GoogleNews-vectors-negative300.bin.gz
...
+Please cite the following works appropriately upon the usage of this repository:
+
+    @inproceedings{adhikari2019rethinking,
+    title={Rethinking Complex Neural Network Architectures for Document Classification},
+    author={Adhikari, Ashutosh and Ram, Achyudh and Tang, Raphael and Lin, Jimmy},
+    booktitle={NAACL},
+    year={2019}
+    }
+
+    @inproceedings{ram2019docbert,
```
Comment (daemon) [inline README.md:86]: Should have arXiv identifier, e.g., arXiv:123123 (see Google Scholar examples).

**Comment by achyudh** [comment]
Why does this pull request change so many files? It should only touch the README files right?

**Review by daemon — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (Ashutosh-Adhikari) [reply README.md:86]: Will add this in a new PR.

**Pull request closed without merging**